platforms/core-execution/daemon-server-worker/build.gradle.kts

```diff
@@ -45,9 +45,14 @@ dependencies {
     implementation(projects.fileCollections)
     implementation(projects.fileOperations)
     implementation(projects.fileTemp)
+    implementation(projects.hashing)
     implementation(projects.persistentCache)
-    implementation(projects.serviceProvider)
     implementation(projects.serviceRegistryBuilder)
+    implementation(projects.buildOperations)
```

Comment:
Are build operations emitted from worker processes routed through to the daemon?

Otherwise I'm a bit skeptical of this new dependency.

Reply:
`WorkerProcessIsolationProblemsServiceProvider` needs `CurrentBuildOperationRef`